Uno.Gallery/Uno.Gallery.Skia.Gtk/Uno.Gallery.Skia.Gtk.csproj

```diff
@@ -3,6 +3,7 @@
 	<PropertyGroup>
 		<OutputType Condition="'$(Configuration)'=='Release'">WinExe</OutputType>
 		<OutputType Condition="'$(Configuration)'=='Debug' or '$(IsCanaryBranch)'=='true'">Exe</OutputType>
+		<!--MediaPlayerElement works on > net7.0-->
 		<TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I think it would be worth retargeting to .NET 7+ anyway, as .NET Core 3.1 is not useful anymore. @jeromelaban

Reply:
Master is already on net7, @rafael-rosa-knowcode make sure to rebase to get the latest.

Reply:
@jeromelaban Rebased. 
Updated the GTK project to make it work with MPE.